.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+        ports: ["5432:5432"]
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.138.0
```

Comment:
I think you can just use `ruby/setup-ruby@v1`

Reply:
On it! 👍 
Curious though, I thought the bigger the number the better, doesn't it mean that we're using an updated version of ruby action?

Reply:
Understood! So specifying `v1` is way better than specifying a specific version like `v1.138.0` because `v1` means we're getting all the patches & minor updates of the major version `v1` while `v1.138.0` means that we're stuck with the same version without getting the latest patches & minor updates unless specified.

Cool! 😎